backend/app/controllers/spree/admin/stores_controller.rb

```diff
@@ -58,7 +58,7 @@ def update
           redirect_to admin_stores_path
         else
           flash[:error] = "#{Spree.t('store_errors.unable_to_update')}: #{@store.errors.full_messages.join(', ')}"
-          render :edit
+          redirect_to edit_admin_store_path(@store)
```

Comment:
```suggestion
          redirect_to spree.edit_admin_store_path(@store)
```

It's always safer to use `spree` instance to scope the routes to Spree engine routes